ui/src/router/index.js

```diff
@@ -18,11 +21,6 @@ const routes = [
     name: "Settings",
     component: Settings,
   },
-  {
-    path: "/logs",
-    name: "Logs",
-    component: Logs,
-  },
```

Comment:
Why Logs view has been removed?

Reply:
No longer used in modern UI, link to logging interface is used instead